pkg/doc/util/jwkkid/kid_creator.go

```diff
@@ -52,6 +52,8 @@ func CreateKID(keyBytes []byte, kt kms.KeyType) (string, error) {
 
 		return x25519KID, nil
 	case kms.ED25519Type: // go-jose JWK thumbprint of Ed25519 has a bug, manually build it and build its resulting KID.
+		// TODO remove `case kms.ED25519Type` when go-jose fixes Ed25519 JWK thumbprint.
```

Comment:
go jose already implemented the fix, need to update the dep and remove this case fully since j.Thumbprint() below will handle the correct ED25519 type

Reply:
What go-jose version added the fix? If we update, do we want to update to latest?

Reply:
It dosen't hurt to be at latest.. but I'll check the version when it was updated

Reply:
Weird, I could swear to have seen that they fixed it. It doesn't seem to be fixed yet: https://github.com/square/go-jose/blob/v2/jwk.go#L335

it still uses a bad ed25519 jwk string for thumbnail (missing `"` before `x`)

Reply:
ahhh... the repo name is different: github.com/go-jose/go-jose

see the fix here: https://github.com/go-jose/go-jose/blob/v3/jwk.go#L341

need to update the go-jose to this new repo instead, the newer v3 has the fix.
square/go-jose is the legacy repo